src/components/itemContextMenu.js

```diff
@@ -99,7 +99,7 @@ import toast from './toast/toast';
             });
         }
 
-        if (!restrictOptions) {
+        if (options.EnableCollectionManagement && !browser.tv) {
```

Comment:
This is still including the "Add to playlist" menu item as well. :wink:

Reply:
I removed the admin check alltogether. There is no reason why any of those items should only be available for admins.
Replaced it by just checking for `browser.tv` and disabling the entries if we're on TV.